ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -836,7 +861,7 @@ function SignatureIntegrationForm({
                                         <Checkbox
                                             label="Enable transparency log validation"
                                             id="transparencyLog.enabled"
-                                            isChecked={values.transparencyLog.enabled}
+                                            isChecked={values?.transparencyLog?.enabled}
```

Comment:
Observation, not necessarily a change request:
* One side of coin: `values?.whatever` is needed only if values might have `null` or `undefined` falsy value
    That is, `values.whatever` will just have value (pardon pun) of `null` (or `undefined` if it is absent)
* Other side of coin: because `transparencyLog` might have `null` or `undefined` falsy value, `?.` is indeed needed for `enabled` and so on